resources/inference_service.py

```diff
@@ -0,0 +1,87 @@
+from ocp_resources.resource import NamespacedResource
+
+from utilities.constants import KSERVE_API_GROUP
+
+
+# TODO: Move this to openshift-python-wrapper once we are confident
+class InferenceService(NamespacedResource):
+    """
+    InferenceService object
+    """
+
+    api_group = KSERVE_API_GROUP
+
+    class DeploymentMode:
+        """
+        DeploymentMode object
+        """
+
+        MODEL_MESH = "ModelMesh"
+
+    def __init__(
+        self,
+        name=None,
```

Comment:
`name`, `namespacem`, `yam_file` and `client` are part of `NamespaceResource` and should be in `kwargs`.

Reply:
done